test/e2e/scheduler_predicates.go

```diff
@@ -118,7 +118,7 @@ func verifyResult(c *client.Client, podName string, ns string) {
 			"involvedObject.kind":      "Pod",
 			"involvedObject.name":      podName,
 			"involvedObject.namespace": ns,
-			"source":                   "scheduler",
+			"source":                   api.DefaultSchedulerName,
```

Comment:
I think we shouldn't change this one, since this is about 1.0 branch, which didn't change.